### PR TITLE
Benefits Questionnaire - Miltary Service: Refactor question order

### PR DESCRIPTION
**Pull request by ghost**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

If the folder you changed contains a `manifest.json`, search for its `entryName` in the content-build [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) (the `entryName` there will match).

If an entry for this folder exists in content-build and you are:
1. **Deleting a folder**: Delete the application entry in [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) and merge that PR **before** this one
- _Add the link to your merged content-build PR here_

2. **Renaming or moving a folder**: Update the entry in the [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json), but do not merge it until your vets-website changes here are merged. The content-build PR must be merged immediately after your vets-website change is merged in to avoid CI errors with content-build (and Tugboat).

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [ ] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

This PR updates the currently disabled benefit eligibility questionnaire military service page question order and key names to something more appropriate.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#0000
- _Link to previous change of the code/bug (if applicable)_
department-of-veterans-affairs/vets-website#0000
- _Link to epic if not included in ticket_
department-of-veterans-affairs/va.gov-team#0000

## Testing done

Local testing.

## Screenshots

Screen Shot 2024-08-09 at 8 27 32 AM
Screen Shot 2024-08-09 at 8 27 42 AM
Screen Shot 2024-08-09 at 8 27 54 AM

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/benefit-eligibility-questionnaire/pages/militaryService.js [hunk 1]
```diff
@@ -1,20 +1,93 @@
...
     properties: {
-      militaryService: {
+      militaryServiceCurrentlyServing: {
+        type: 'string',
+        enum: ['Yes', 'No'],
+      },
+      militaryServiceTotalTimeServed: {
+        type: 'string',
+        enum: [
+          'Less than 90 Days',
```
Comment (Andrew565) [inline src/applications/benefit-eligibility-questionnaire/pages/militaryService.js:83]: 90 days

File: src/applications/benefit-eligibility-questionnaire/pages/militaryService.js [hunk 2]
```diff
@@ -1,20 +1,93 @@
...
+          'More than 90 days but less than 1 year',
+          'More than 1 year but less than 2 years',
+          'More than 2 year but less than 3 years',
```
Comment (Andrew565) [inline src/applications/benefit-eligibility-questionnaire/pages/militaryService.js:86]: 2 years

File: src/applications/benefit-eligibility-questionnaire/pages/militaryService.js [hunk 3]
```diff
@@ -1,20 +1,93 @@
...
+        widgetProps: {
+          upTo90days: { militaryService: 'Less than 90 days' },
+          upTo1yr: {
+            militaryService: 'More than 90 days but less than 1 year',
+          },
+          upTo2yr: {
+            militaryService: 'More than 1 year but less than 2 years',
+          },
+          upTo3yr: {
+            militaryService: 'More than 2 year but less than 3 years',
```
Comment (Andrew565) [inline src/applications/benefit-eligibility-questionnaire/pages/militaryService.js:46]: 2 years

**Review by Andrew565 — DISMISSED** [review]
Approved but suggest fixing typos.

**Review by vetskrieg — CHANGES_REQUESTED** [review]
Can these fields be moved to separate pages within a single chapter? I think multiple sub-pages can be included within the military service chapter of form.js.

**Review by vetskrieg — COMMENTED** [review]
I think you forgot to commit your form.js file. Does it look like this?

`chapter2: {
      title: 'Service',
      pages: {
        militaryServiceCurrentlyServing: {
          path: 'military-service-1',
          title: 'Military Service',
          uiSchema: militaryServiceCurrentlyServing.uiSchema,
          schema: militaryServiceCurrentlyServing.schema,
        },
        militaryServiceTotalTimeServed: {
          path: 'military-service-2',
          title: 'Military Service',
          uiSchema: militaryServiceTotalTimeServed.uiSchema,
          schema: militaryServiceTotalTimeServed.schema,
        },
        militaryServiceCompleted: {
          path: 'military-service-3',
          title: 'Military Service',
          uiSchema: militaryServiceCompleted.uiSchema,
          schema: militaryServiceCompleted.schema,
        },
      },
    },`

**Review by vetskrieg — APPROVED** [review]

**Review by asg5704 — APPROVED** [review]
No Identity files touched. Approving.

Note - in the future you could probably close the Pull Request and re-open it as you'll have to get all those people to review to get this passing.

**Pull request closed without merging**